pipeline/stage03_trigger_detection/scripts/minima.py

```diff
@@ -183,13 +183,16 @@ def plot_minima(asig, event, channel, maxima_threshold_window,
 
     block = load_neo(args.data)
     asig = block.segments[0].analogsignals[0]
+    
+    if args.maxima_threshold_window is None or args.maxima_threshold_window > asig.t_stop - asig.t_start:
+         args.maxima_threshold_window = asig.t_stop - asig.t_start
```

Comment:
`args.maxima_threshold_window` is a float value for a time period in seconds, whereas `asig.t_stop` and `asig.t_start` are quantity values. So, before comparing them or assigning one to the other, the quantity values need to be rescaled to seconds and converted to floats.